src/scss/components/dropdown-select/_config.scss

```diff
@@ -4,6 +4,10 @@
 
 /// ========== MIXINS ========== ///
 @import "mixins";
+@import "./../form/mixins/fieldName";
```

Comment:
please remove `./` from beginning of import path (it is redundant)

Reply:
Changes have been done, please review.